babel.config.ts

```diff
@@ -18,14 +18,11 @@ const config: TransformOptions = {
         corejs: '3.27',
       },
     ],
-    '@babel/preset-typescript',
+    // TODO: Remove allowDeclareFields when we upgrade to Babel 8
+    ['@babel/preset-typescript', {allowDeclareFields: true}],
   ],
   overrides: [],
-  plugins: [
-    '@emotion/babel-plugin',
-    '@babel/plugin-transform-runtime',
-    '@babel/plugin-transform-class-properties',
```

Comment:
Do we need to also remove this as a dependency?

Reply:
I think I looked and it's not listed

Reply:
For future reference: I think we should add a rule to never use a dependency of a dependency directly in the project.